client/my-sites/themes/theme-options.js

```diff
@@ -170,10 +168,10 @@ export const connectOptions = connect(
 		let mapAction;
 
 		if ( site ) {
-			// TODO (@ockham): Change actions to use siteId.
-			mapAction = action => ( t ) => action( t, site, source );
+			// TODO (@ockham): Change actions to use siteId. (@budzanowski) activateTheme is using only IDs
```

Comment:
I think we can drop this comment altogether now.

One thing I'd like us to do (as part of this PR) is change `connectOptions` to accept `siteId` instead of a `site` object.